Pipfile

```diff
@@ -8,6 +8,9 @@ name = "pypi"
 aiodns = "*"
 aiohttp = "<2.3.0,>=2.0.0"
 websockets = ">=4.0,<5.0"
+"beautifulsoup4" = "*"
+requests-html = "*"
```

Comment:
It looks like you've edited this file by hand. This is bad practise, but at least you've updated the lockfile as well.

Please be careful about using `requests` - it's not asynchronous.

Reply:
I am 100% sure that I did not update the Pipfile by hand. I used pipenv install requests-html

Reply:
Okay. Either way, we've made a statement about the wikipedia module on Discord, and the same applies to Requests. Requests does not support asyncio, and you're going to want to avoid any solution that doesn't.